src/mcp-server.ts

```
@@ -111,173 +140,248 @@ export class AzureDevOpsMCPServer {
+        )
+        break
       case 'open':
-        const newItems = await this.db.getWorkItemsByStateForUsers('New', this.userEmails);
-        const activeItems = await this.db.getWorkItemsByStateForUsers('Active', this.userEmails);
-        const inProgressItems = await this.db.getWorkItemsByStateForUsers('In Progress', this.userEmails);
-        items = [...newItems, ...activeItems, ...inProgressItems];
-        break;
+        const newItems = await this.db.getWorkItemsByStateForUsers(
+          'New',
+          this.userEmails,
+        )
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)